tests/openwisp2/settings.py

```diff
@@ -48,9 +48,16 @@
     'drf_yasg',
     'django_extensions',
     'openwisp2.integrations',
+    'djangosaml2',
 ]
 
-LOGIN_REDIRECT_URL = 'admin:index'
+# Bug in djangosaml2: https://git.io/JGUb9
+LOGIN_REDIRECT_URL = '/admin/index'
+
+AUTHENTICATION_BACKENDS = (
+    'django.contrib.auth.backends.ModelBackend',
```

Comment:
let's take advantage to switch to `openwisp_users.backends.UsersAuthenticationBackend` which will be the new default, can you make sure to also document this please?